### ISSUE TITLE
Run triton python server with custom conda-pack

### ISSUE THREAD

**Issue by jeanbaptiste-brasselet** [kubeflow/kfserving#1761]
I am new to this, I have been able to deploy the add_sub example from there : https://github.com/triton-inference-server/python_backend

But I want to be able to run a model with some dependencies not installed by default, the documentation says I can add a file using conda-pack.

The solution I tried is to add the file next to my model and let the storage-initializer do the copy but in the end it does not work because the version used by KFserving is this one (NVIDIA Release 20.10) at least on my setup and the feature is more recent than that. So I was thinking maybe there is a way to update the version used but I was not able to do it. I tried to update the kfserving.yaml (triton/image part) but nothing has changed.

And I am not even sure it's the correct way to do it because it seems like a hack to add it next to the models and all.

Any help would be gladly accepted, thank in advance.

**Comment by yuzisun** [comment]
@deadeyegoodwin you want to send a PR to update the default version to the latest version?

@jeanbaptiste-brasselet you can also [overwrite the version](https://github.com/kubeflow/kfserving/blob/master/docs/samples/v1beta1/triton/torchscript/torchscript.yaml#L9) on the inference service yaml

**Comment by jeanbaptiste-brasselet** [comment]
Thanks it works :) even if now I have an issue with triton itself .... It start by default SageMaker in the latest version, they did a fix but not yet deployed.

To fix the problem I would need to be able to add a flag --allow-sagemaker=false when the docker run,  is it something possible ?

**Comment by yuzisun** [comment]
@jeanbaptiste-brasselet Yes, I think you can pass that flag on the args section from the inference service yaml

**Comment by deadeyegoodwin** [comment]
We are releasing Triton 21.08 soon, after that I will update to use that version.
